projects/observability/src/pages/apis/api-detail/api-detail-breadcrumb.resolver.ts

```diff
@@ -51,13 +47,18 @@ export class ApiDetailBreadcrumbResolver implements Resolve<Observable<Breadcrum
     );
   }
 
-  private getParentBreadcrumb(api: ApiBreadcrumbDetails, parentEntityMetadata?: EntityMetadata): Breadcrumb[] {
+  private getParentBreadcrumbs(api: ApiBreadcrumbDetails, parentEntityMetadata?: EntityMetadata): Breadcrumb[] {
     return parentEntityMetadata !== undefined
       ? [
           {
             label: api.parentName,
             icon: parentEntityMetadata?.icon,
             url: parentEntityMetadata?.detailPath(api.parentId)
+          },
+          {
+            label: 'Endpoints',
```

Comment:
Should we maintain the unclickable endpoints breadcrumb when there is no parent? That was removed here, unsure if that was intentional.

Reply:
That was intentional. It did not make sense to have a breadcrumb of the format `Services / Endpoints / <Endpoint Name>`. Without the parent name, the 'endpoints' section is a non existent page. @aaron-steinfeld